src/components/Layout/PageLayout.js

```diff
@@ -1,11 +1,12 @@
-import React, { useEffect } from "react";
+import React, { useContext, useEffect } from "react";
 import PropTypes from "prop-types";
 import { useMobile } from "../../utils/media-queries";
-import useComponentController from "../../hooks/useComponentController";
 import DesktopLayout from "./DesktopLayout";
 import MobileLayout from "./MobileLayout";
 import BaseLayout from "./BaseLayout";
 
+const DefaultContext = React.createContext();
```

Comment:
import the default context from the useComponentController file (https://github.com/NIAEFEUP/nijobs-fe/blob/develop/src/hooks/useComponentController.js) It's the same for now, but if we ever need to change it, it's better to use the same